services/config/default.json

```diff
@@ -61,12 +63,14 @@
     "MAX_FILE_SIZE": 104857600
   },
   "REDIS": {
-    "HOST": "localhost",
+    "HOST": "redis",
     "PORT": "6379",
-    "PASSWORD": ""
+    "PASSWORD": "redis"
   },
   "EMAIL": {
-    "DEFAULT_ADMINS": [""]
+    "DEFAULT_ADMINS": [
+      ""
+    ]
   },
```

Comment:
These changes could break someone's local setup. Not an issue to fix but it could be avoided

Reply:
I'll make a post in #dev before I merge